packaging/aznfs/DEBIAN/postinst

```diff
@@ -11,6 +11,62 @@ set -e
 FLAG_FILE="/tmp/.update_in_progress_from_watchdog.flag"
 CONFIG_FILE="/opt/microsoft/aznfs/data/config"
 AUTO_UPDATE_AZNFS="false"
+INSTALLED_STUNNEL_VERSION="0"
+REQUIRED_STUNNEL_VERSION="5.73"
+
+cleanup_stunnel_files()
+{
+    cd -
+    rm -rf /tmp/$stunnel_dir
+    rm -f /tmp/stunnel-5.73.tar.gz
+}
+
+install_stunnel()
+{
+    # Install stunnel package from source
+    wget https://www.stunnel.org/downloads/stunnel-5.73.tar.gz -P /tmp
+    if [ $? -ne 0 ]; then
+        echo "Failed to download stunnel source code. Please install stunnel and try again."
+        exit 1
+    fi
+
+    tar -xvf /tmp/stunnel-5.73.tar.gz -C /tmp
+    if [ $? -ne 0 ]; then
+        echo "Failed to extract stunnel tarball. Please install stunnel and try again."
+        cd -
+        rm -f /tmp/stunnel-5.73.tar.gz
+        exit 1
+    fi
+
+    stunnel_dir=$(tar -tf /tmp/stunnel-5.73.tar.gz | head -n 1 | cut -f1 -d'/')
+
+    cd /tmp/$stunnel_dir
+    ./configure
+    if [ $? -ne 0 ]; then
+        echo "Failed to configure the build. Please install stunnel and try again."
+        cleanup_stunnel_files
+        exit 1
+    fi
+
+    make
+    if [ $? -ne 0 ]; then
+        echo "Failed to build stunnel. Please install stunnel and try again."
+        cleanup_stunnel_files
+        exit 1
+    fi
+
+    sudo rm -f /bin/stunnel
+
+    make install
+    if [ $? -ne 0 ]; then
+        echo "Failed to install stunnel. Please install stunnel and try again."
+        cleanup_stunnel_files
+        exit 1
+    fi
+
+    cleanup_stunnel_files
+    sudo ln -sf /usr/local/bin/stunnel /bin/stunnel
```

Comment:
this is not needed

Reply:
I took another look on the stunnel installation. If there already is a stunnel installation, the binary is in **/bin/stunnel**. The stunnel from the package is installed in **/usr/local/bin/stunnel**. So after installing from source code, we should create the symlink for the stunnel to work fine.

I do need this line, otherwise the link doesn't work fine:

azureuser@sroghanchiubuntu22:~$ which stunnel
/usr/local/bin/stunnel
azureuser@sroghanchiubuntu22:~$ stunnel -version
-bash: /usr/bin/stunnel: No such file or directory